tests/src/io/openliberty/tools/eclipse/test/it/LibertyPluginSWTBotGradleTest.java

```diff
@@ -158,21 +167,43 @@ public static final void validateBeforeTestRun() {
         List<String> runAsMenuItems = runAsMenu.menuItems();
         Assertions.assertTrue(runAsMenuItems != null && !runAsMenuItems.isEmpty(),
                 "The runAs menu associated with project: " + GRADLE_APP_NAME + " is null or empty.");
-        int foundItems = 0;
+        int foundRunAsItems = 0;
 
         for (String expectedItem : runAsShortcuts) {
             for (String item : runAsMenuItems) {
                 if (item.contains(expectedItem)) {
-                    foundItems++;
+                    foundRunAsItems++;
                     break;
                 }
             }
         }
 
-        Assertions.assertTrue(foundItems == runAsShortcuts.length,
+        Assertions.assertTrue(foundRunAsItems == runAsShortcuts.length,
                 "The runAs menu associated with project: " + GRADLE_APP_NAME
                         + " does not contain one or more expected entries. Expected number of entries: " + runAsShortcuts.length
-                        + "Found entry count: " + foundItems + ". Found menu entries: " + runAsMenuItems);
+                        + "Found entry count: " + foundRunAsItems + ". Found menu entries: " + runAsMenuItems);
```

Comment:
Our existing tests should cover this since we are already testing "Start" and "Start..." from the "Debug As" menu:

`testStartWithDebugAsShortcut` and `testStartWithCustomDebugAsConfig`

Reply:
There are gaps in our dashboard testing, however. Today I dont think we test any debug options there. I'll see about adding that in.

Reply:
Ok - tests added and passing.